`pyvex` fails to build on macos 10.14. Manages to build `libvex` with clang, but then tries to resort to gcc for `libpyvex`:

```
Building libpyvex
gcc -c -g -O2 -Wall -shared -fPIC -std=c99 -I "/private/tmp/nix-build-python3.8-pyvex-9.0.6281.drv-0/pyvex-9.0.6281/vex/pub" pyvex.c
gcc -c -g -O2 -Wall -shared -fPIC -std=c99 -I "/private/tmp/nix-build-python3.8-pyvex-9.0.6281.drv-0/pyvex-9.0.6281/vex/pub" logging.c
gcc -c -g -O2 -Wall -shared -fPIC -std=c99 -I "/private/tmp/nix-build-python3.8-pyvex-9.0.6281.drv-0/pyvex-9.0.6281/vex/pub" analysis.c
/bin/sh: gcc: command not found
make: *** [Makefile:40: pyvex.o] Error 127
make: *** Waiting for unfinished jobs....
/bin/sh: gcc: command not found
make: *** [Makefile:40: logging.o] Error 127
/bin/sh: gcc: command not found
make: *** [Makefile:40: analysis.o] Error 127

```

--- a/pkgs/development/python-modules/ailment/default.nix
+++ b/pkgs/development/python-modules/ailment/default.nix
@@ -7,14 +7,14 @@
 
 buildPythonPackage rec {
   pname = "ailment";
-  version = "9.0.5903";
+  version = "9.0.6281";
   disabled = pythonOlder "3.6";
 
   src = fetchFromGitHub {
     owner = "angr";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-75Ul9JfMFYv3AfBlgmer6IDyfgOAS4AdXexznoxi35Y=";
+    sha256 = "sha256-IFUGtTO+DY8FIxLgvmwM/y/RQr42T9sABPpnJMILkqg=";
   };
 
   propagatedBuildInputs = [ pyvex ];

--- a/pkgs/development/python-modules/archinfo/default.nix
+++ b/pkgs/development/python-modules/archinfo/default.nix
@@ -7,13 +7,13 @@
 
 buildPythonPackage rec {
   pname = "archinfo";
-  version = "9.0.5903";
+  version = "9.0.6281";
 
   src = fetchFromGitHub {
     owner = "angr";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-4e+ZGIt/ouZj5rsmaVxUrz8gAq4Yq2+Qx4jdOojB4Sw=";
+    sha256 = "sha256-ZO2P53RdR3cYhDbtrdGJnadFZgKkBdDi5gR/CB7YTpI=";
   };
 
   checkInputs = [

--- a/pkgs/development/python-modules/claripy/default.nix
+++ b/pkgs/development/python-modules/claripy/default.nix
@@ -13,14 +13,14 @@
 
 buildPythonPackage rec {
   pname = "claripy";
-  version = "9.0.5903";
+  version = "9.0.6281";
   disabled = pythonOlder "3.6";
 
   src = fetchFromGitHub {
     owner = "angr";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-NIKWUx1VT5TjnuqppuT6VzwNRwcBLc0xI5k3F2Nmj8A=";
+    sha256 = "sha256-gvo8I6LQRAEUa7QiV5Sugrt+e2SmGkkKfsGn/IKz+Mk=";
   };
 
   # Use upstream z3 implementation

